cmd/livepeer/livepeer.go

```diff
@@ -581,6 +600,32 @@ func main() {
 			}
 		}
 
+		if *redeemer {
+			r, err := server.NewRedeemer(
+				n.Eth.Account().Address,
+				n.Eth,
+				pm.NewSenderMonitor(recipientAddr, n.Eth, senderWatcher, timeWatcher, n.Database, cleanupInterval, smTTL),
+			)
+			if err != nil {
+				glog.Errorf("Unable to create redeemer: %v", err)
+				return
+			}
+
+			uri, err := url.Parse(*httpAddr)
```

Comment:
We could do something like:

```
*httpAddr = defaultAddr(*httpAddr, "127.0.0.1", RpcPort)
```

so the user can pass in values like `:<PORT>` or `<IP>:` with the defaults used for the missing fields. We do this in other places in this file for `httpAddr`, but I don't think we do it before this code executes. I think `defaultAddr` should also just return the passed in address if all fields are filled in so it should be safe to use multiple times on the same address, but this is worth double checking.

Reply:
fixed in c0f617298df0d7c9472e110eb0d33e06f5cc99bf